bin/schemats.ts

```diff
@@ -18,10 +18,12 @@ let argv: any = yargs
     .alias('c', 'conn')
     .nargs('c', 1)
     .describe('c', 'database connection string')
-    .demand('t')
```

Comment:
That works too. Just need to document this behaviour in `schemats --help` output.